x-pack/libbeat/autodiscover/providers/aws/elb/_meta/fields.yml

```diff
@@ -0,0 +1,84 @@
+- key: elb_listener
+  title: "ELB Listener"
+  description: >
+    ELB Listeners
+  short_config: false
+  release: experimental
+  fields:
+    - name: elb_listener
```

Comment:
Should we try to map some of the fields below to ECS? Would that be more confusing then valuable? Undecided.

Reply:
I think we should wait. I'm not sure if we're at the point where we want every AWS API mapped into ECS. EC2, sure. But ELBs feel too specific.

Would be curious to here from @MikePaquette and @webmat.

Reply:
Most of the stuff here is very AWS-specific (SGs, VPC id, etc). So most of them are fine as custom fields, IMO. Even the stuff that's kind of network-centric follows different semantics (`ip_address_type` and `type`), and I don't think we should try to fit those in ECS.

I do think it would make sense to at fill out the following, though:

* `destination.port`
* `destination.address` (from `host`)
* `destination.ip`

If Heatbeat knows its own IP and outgoing port, perhaps fill `source.[ip|port]` as well.

Reply:
Well, since most of these ECS fields will be filled by heartbeat or whatever other beat uses these I'm averse to starting to add extra `destination`  fields, I think users would find it confusing. Additionally, I'd argue that `destination` isn't a great fit in the first place here.